### ISSUE TITLE
Benchmark deferred loading separately

### ISSUE THREAD

**Issue by rossmacarthur** [rossmacarthur/zsh-plugin-manager-benchmark#2]
- zinit using its turbo mode
- other plugin managers using [romkatv/zsh-defer](https://github.com/romkatv/zsh-defer)

**Comment by aamehedi** [comment]
+1

**Comment by Maple38** [comment]
Yeah, this repo is honestly pretty misleading. If I didn't know better, I'd look at the charts and assume Zinit is slow, when that's very far from the truth. Upon testing, my Zinit configuration takes ~60ms to load, and that's with dozens of plugins installed.
